.github/workflows/publish-macros-dry-run.yml

```diff
@@ -0,0 +1,14 @@
+name: PublishMacrosDryRun
+
+on: workflow_dispatch
+
+jobs:
+  publish_macros_dry_run:
+    name: PublishMacrosDryRun
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: PublishDryRun-Macros
+        run: cargo publish -p rs-matter-macros --dry-run
```

Comment:
One could make this a [workflow input](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-onworkflow_dispatchinputs) so that you didn't have two separate workflow files. Would be more work now but ideally less maintenance in the future.

Reply:
I would keep it separate. This way it is symmetric with the separate workflows for the actual publishment of `rs-matter` and `rs-matter-macros`. The code duplication is not so big IMO.

Reply:
I would mark this as resolved, as I'm hesitant to do this change. Besides the earlier argument - also I think w.r.t. publishing (including dry-run) - the simpler the better.